src/extension.ts

```diff
@@ -64,9 +65,10 @@ export function activate(context: vscode.ExtensionContext) {
   new OneExplorer(context, logger);
 
   // ONE view
-  let refreshCompiler = vscode.commands.registerCommand('onevscode.refresh-toolchain', () => {
-    console.log('refresh-toolchain: NYI');
-  });
+  const toolchainProvier = new ToolchainProvider();
+  vscode.window.registerTreeDataProvider('ToolchainView', toolchainProvier);
```

Comment:
The registered treeDataProvider must be subscribed to be disposed after the extension's expiration.
```suggestion
  context.subscriptions.push(
    vscode.window.registerTreeDataProvider('ToolchainView', toolchainProvier));
```

FYI, https://github.com/Samsung/ONE-vscode/issues/643